src/components/PersonLink.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Link, useLocation, useResolvedPath } from 'react-router-dom';
+
+import classNames from 'classnames';
+
+import '../App.scss';
+import { Person } from '../types';
+
+type Props = {
+  personName: string,
+  people: Person[],
+};
+
+export const PersonLink: React.FC<Props> = ({ personName, people }) => {
+  const findPerson = people.find(human => human.name === personName);
+
+  const location = useLocation();
+  const parentPath = useResolvedPath('../').pathname;
+
+  return (
+    findPerson ? (
+      <Link
+        to={{
+          pathname: parentPath + findPerson.slug,
```

Comment:
what if findPerson will be undefined?

Reply:
@Denchepornyuk  its a findPerson ? statement above, so this bloc of code will run only if we have findPerson ))